packages/db/package.json

```diff
@@ -9,8 +9,7 @@
     "db-up": "docker compose up -d",
     "db-generate": "prisma generate",
     "db-push": "prisma db push",
-    "db-down": "docker compose down",
-    "typecheck": "tsc --noEmit"
```

Comment:
this is a bit of a weird one, I don't think we need to type check here but if we would, we'd need to make it `pnpm prisma generate && tsc --noEmit` since db-generate isn't an external script

Reply:
Why not mark the `db-generate` as a dependency for the typecheck?

Feels like almost all our types depend on this command.

Reply:
Oh I see that you did that. And it's still not guaranteed to run before?

What does the `dependOn` do then? 🤔